pkg/proxy/iptables/proxier.go

```diff
@@ -591,6 +593,49 @@ func (proxier *Proxier) OnEndpointSlicesSynced() {
 	proxier.syncProxyRules()
 }
 
+// OnNodeAdd is called whenever creation of new node object
+// is observed.
+func (proxier *Proxier) OnNodeAdd(node *v1.Node) {
+	if node.Name != proxier.hostname {
+		klog.Errorf("BAD NEWS: Got a watch event for a node that isn't me: %s", node.Name)
```

Comment:
Thoughts on this instead: 

```
klog.Errorf("Received a watch event for a node %s that doesn't match the current node %v", node.Name, proxier.hostname)
```

I think bad news is implied w/ klog.Errorf

Reply:
Yes, will change to that.